src/test/ui/feature-gates/feature-gate-weak-into-raw.stderr

```diff
@@ -0,0 +1,21 @@
+error[E0658]: use of unstable library feature 'weak_into_raw'
```

Comment:
and therefore this...

...is unnecessary for library features. :)

Reply:
Seems like I should just drop the whole first commit, then.